src/styles/StyleUtils.ts

```diff
@@ -606,6 +606,16 @@ function getFontFamilyMonospace({fontStyle, fontWeight}: TextStyle): string {
 
     return italicBold || bold || italic || fontFamily.MONOSPACE;
 }
+/**
+ * Returns the font size for the HTML code tag renderer.
+ */
+function getCodeFontSize(isInsideH1:Boolean, textStyle :TextStyle) {
+    if (isInsideH1) {
+      return 15;
+    } else {
+      return textStyle.fontSize;
+    }
+  }
```

Comment:
It may be better to remove the fontSize from the textStyle and just use 13 directly.

In `styles.ts` we should state that the font size is calculated by this function